### PR TITLE
My branch

### LINKED ISSUE #1: Project files
-create components folder
-create pages folder

### PR DESCRIPTION
**Pull request by akuutsang**
this pull request closes #1

### CODE REVIEW

File: rportfolio/src/components/nav/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  return (
+    <div className="navSection">
+        <div className="NavLeft">
+          <div className="logo">Akutsang</div>
+        </div>
+        <div className="navRight">
+            <div className="home">Home</div>
+            <div className="about">About</div>
+            <div className="skills">Skills</div>
+            <div className="contact">Contact</div>
```
Comment (antoHero) [inline rportfolio/src/components/nav/Nav.jsx]: Your nav bar items are supposed to be links correct? Or you don't intend to navigate users to the respective pages/sections. If yes, why and why not routes?

File: rportfolio/src/index.js [hunk 2]
```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
```
Comment (antoHero) [inline rportfolio/src/index.js]: Do you still have this file? I can't find it. It may cause unexpected errors, please remove this line

**Review by antoHero — CHANGES_REQUESTED** [review]
Great job Winnie, it's great to finally see you work on the React version of your portfolio.  I would have loved to see the UI in your READMe file, kindly update it and attend to the comments. 🚀🚀

File: rportfolio/src/components/nav/Nav.jsx (hunk 1 above)
Comment (akuutsang) [reply rportfolio/src/components/nav/Nav.jsx]: Link has been added sir.

File: rportfolio/src/index.js (hunk 2 above)
Comment (akuutsang) [reply rportfolio/src/index.js]: i have removed the unused file as well

**Pull request merged**